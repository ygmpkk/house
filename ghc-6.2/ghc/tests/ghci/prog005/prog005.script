:unset +s
:unset +t
:set -package lang -fglasgow-exts
:l Parser
parser (lexer "a>b")
