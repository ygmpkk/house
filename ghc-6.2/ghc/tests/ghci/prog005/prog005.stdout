   ___         ___ _
  / _ \ /\  /\/ __(_)
 / /_\// /_/ / /  | |      GHC Interactive, version 5.03, for Haskell 98.
/ /_\\/ __  / /___| |      http://www.haskell.org/ghc/
\____/\/ /_/\____/|_|      Type :? for help.

Loading package std ... linking ... done.
Loading package lang ... linking ... done.
Ok, modules loaded: Parser.
2
Leaving GHCi.
